### PR TITLE
fetch routes to prerender instead of crawling

### PR DESCRIPTION
**Pull request by sjoerdbeentjes**
- Removes config for crawling
- Adds function that fetches all routes that should be prerendered
- Implements this function in the `prerender:routes` nuxt hook

### CODE REVIEW

File: src/scripts/fetch-routes.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        path: '/contact/'
+    },
+    {
+        queryOperation: 'allJobs',
+        path: '/jobs/'
+    }
+]
+
+// fetches data from DatoCMS using the GraphQL API
+const fetchDatoQuery = async ({ query, variables = {} }) => {
```
[the remark this answers is not in the document]
Comment (sjoerdbeentjes) [reply src/scripts/fetch-routes.js]: https://github.com/voorhoede/voorhoede-website/issues/601

**Comment by sjoerdbeentjes** [comment]
> I trust Frank's review! Just one question I have is I see you added some comments with JSDoc types, why not write the fetch-routes script in TS?

Yeah nice one. I've rewritten it to typescript: [c7ada32](https://github.com/voorhoede/voorhoede-website/pull/587/commits/c7ada322d0d07b58d732d4c7fcb8ae49c1dd3bfd)

File: src/scripts/fetch-routes.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Siilwyn) [reply src/scripts/fetch-routes.js]: Thanks!

File: src/scripts/fetch-routes.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  skip: number;
+}) => {
+  return fetchDatoQuery({
+    query: `
+                query ${operation}($skip: IntType, $locale: SiteLocale) {
+                    ${operation}(first: 100, skip: $skip, locale: $locale) {
+                        slug
+                    }
+                }
+            `,
```
Comment (Siilwyn) [inline src/scripts/fetch-routes.ts:103]: Pretty big indendation?

File: src/scripts/fetch-routes.ts [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  operation: string;
+  locale: string;
+}) => {
+  return fetchDatoQuery({
+    query: `
+            query Meta ($locale: SiteLocale) {
+                _${operation}Meta(locale: $locale) {
+                    count
+                }
+            }
```
Comment (Frankwarnaar) [inline src/scripts/fetch-routes.ts:125]: Just like Selwyn mentioned in the other comment, indentation is a bit big here as well

**Review by Frankwarnaar — APPROVED** [review]
Aside from the indentation, looks good to merge to me!

**Review by Siilwyn — APPROVED** [review]

File: src/scripts/fetch-routes.ts (hunk 2 above)
Comment (sjoerdbeentjes) [reply src/scripts/fetch-routes.ts]: Don't know how that happened, it's fixed now [a54563c](https://github.com/voorhoede/voorhoede-website/pull/587/commits/a54563ccca63dae07409b35419a8af6eefa9ee67)

**Pull request merged**